taxonomies/food/categories.txt

```diff
@@ -53369,13 +53363,15 @@ nl: Yoghurtdranken met vanille
 pt: Bebidas de iogurte de baunilha
 ciqual_food_code:en: 19508
 
-< en:Flavoured yogurt drinks
+< en:Flavoured yogurt drinks with sugar
 en: Chocolate yoghurt drinks
 fr: Yaourts à boire goût chocolat
 nl: Yoghurt dranken met chocolade
 
-< en:Flavoured yogurt drinks
+< en:Fermented milk drinks with fruits
```

Comment:
There's a choice to make: it is with fruits, or flavoured with fruits?

Reply:
I assume the category "Fermented milk drinks with fruits" meant with fruit as ingredient or with fruit flavour, but looking at [product in it](https://fr.openfoodfacts.org/facets/categories/Boissons%20lact%C3%A9es%20ferment%C3%A9es%20aux%20fruits), it seems to be messy, with a lot of fruit yogurt, not even drinkable, and with both fruit or fruit flavor.
For my use case, I don't care about this category, but I tried to enhance it by not letting it standalone, without link to other categories. But maybe it was a mistake :-)